openpype/hosts/max/plugins/publish/validate_renderpasses.py

```diff
@@ -43,7 +43,26 @@ def process(self, instance):
     @classmethod
     def get_invalid(cls, instance):
         """Function to get invalid beauty render outputs and
-        render elements
+        render elements.
+
+        1. Checking Render Output Folder matches the name of
+        the current Max Scene
+        e.g. The name of the current Max scene:
+                John_Doe.max
+            The expected render output directory:
+            {root[work]}/{project[name]}/{hierarchy}
+            /{asset}/work/{task[name]}/render/3dsmax/John_Doe/
+
+        2. Checking the image extension(s) of the render output(s)
+          does not match with image format in OP/AYON setting.
+        e.g. The current image format in the setting: png
+            The expected render outputs: Joe_Doe.png
+
+        3. Checking the filename of render element is not ended
+            with the name of render element from the 3dsMax Render
+            Element Manager.
+        e.g. The name of render element: RsCryptomatte
+            The expected filename: {InstanceName}_RsCryptomatte.png
```

Comment:
Would be good to 'shorten' this by making `Checking` just `Check`
I'd also write each sentence as what must be ensured - currently 2) mentions checking if it does not match, but I'd make it "Check it matches" or "Validate it matches" or "Ensure it matches". Currently it's a mixture where 1) describes "check this is the case" then the second says "check this does not match" (which is incorrect wording?). Preferably it's the least 'mixed'.

Also "is not ended" should likely be "does not end with". Here's a quick example:

```suggestion
        1. Check Render Output Folder matches the name of
           the current Max Scene, e.g. 
             The name of the current Max scene:
               John_Doe.max
             The expected render output directory:
               {root[work]}/{project[name]}/{hierarchy}/{asset}/
               work/{task[name]}/render/3dsmax/John_Doe/

        2. Check image extension(s) of the render output(s)
           matches the image format in OP/AYON setting, e.g.
               The current image format in settings: png
               The expected render outputs: John_Doe.png

        3. Check filename of render element ends with the name of
           render element from the 3dsMax Render Element Manager.
           e.g. The name of render element: RsCryptomatte
            The expected filename: {InstanceName}_RsCryptomatte.png
```